src/shared/services/document-management/document-management.service.ts

```diff
@@ -50,14 +58,35 @@ export class DocumentManagementService {
       if (position === documentFieldValue.document_filename.length) {
         fileExtension = '';
       } else if (position >= 0) {
-        fileExtension = documentFieldValue.document_filename.slice(position + 1);
+        fileExtension = documentFieldValue.document_filename.slice(position + 1).toUpperCase();
```

Comment:
Hey @ishOsw,

The `.toUpperCase()` could be what is breaking this. In any case this is breaking the test. You need to come up with a solution that doesn't break what is already there.

Reply:
The test for `should return media viewer data for tif content types` in src/shared/services/document-management/document-management.service.spec.ts  specifically deals with the issue of case sensitivity.